python/verifiers/learnedCorres.py

```diff
@@ -0,0 +1,684 @@
+import os
+
+import cv2
+import numpy as np
+import tensorflow as tf
+
+dirname = os.path.dirname(os.path.abspath(__file__))
+
+from verifiers.VerificationTemplate import VerificationTemplate
+
+class learnedCorres(VerificationTemplate):
```

Comment:
can we add a comment here with a brief description about what this class is to be used for?

Reply:
What about the comment I added below?